doc: boards: cc3220sf_launchxl: Document limitations

vanti opened the pull request:
Adding a section on some features that have yet to be implemented in
the SimpleLink WiFi driver.

Signed-off-by: Vincent Wan <vincent.wan@linaro.org>

vanti committed:
doc: boards: cc3220sf_launchxl: Document limitations

Adding a section on some features that have yet to be implemented in
the SimpleLink WiFi driver.

Signed-off-by: Vincent Wan <vincent.wan@linaro.org>

dbkinder commented on boards/arm/cc3220sf_launchxl/doc/index.rst:
```diff
@@ -271,6 +271,17 @@ example.
+- IPv6: While the hardware supports it, it has yet to be fully implemented
+  in the SimpleLink WiFi driver.
+- static IP address: It is not currently possible to set
+  `CONFIG_NET_CONFIG_SETTINGS` to ``y`` and assign a static IP address to
```
better to link to the documentation for the config options with:
```
:option:`CONFIG_NET_CONFIG_SETTINGS`
```
and 
```
:option:`CONFIG_NET_CONFIG_MY_IPV4_ADDR`
```

dbkinder commented on boards/arm/cc3220sf_launchxl/doc/index.rst:
```diff
@@ -271,6 +271,17 @@ example.
 
+Limitations
+***********
+The following features are not yet supported:
```
better to say which version of Zephyr (in case we forget to update this later):
```suggestion
The following features are not supported in Zephyr v1.14:
```

GAnthony reviewed (approved):
LGTM
Thanks!

vanti commented:
Updated based on comments from @dbkinder. Thanks for the reviews.

Not sure why Shippable keeps failing. This is a doc change but it keeps failing on some assertion in a timer test. I have rebased to master but this doesn't seem to help.

dbkinder reviewed (approved):
Thanks!

GAnthony commented:
recheck
